stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/common/extensions/internal/Context.kt

```diff
@@ -63,18 +63,22 @@ internal val Context.streamThemeInflater: LayoutInflater
 
 internal fun Context.createStreamThemeWrapper(uiMode: UiMode = ChatUI.uiMode): Context {
     val typedValue = TypedValue()
-    val configuredContext = configureUiMode(uiMode)
 
     return when {
-        configuredContext.theme.resolveAttribute(R.attr.streamUiValidTheme, typedValue, true) -> configuredContext
-        configuredContext.theme.resolveAttribute(R.attr.streamUiTheme, typedValue, true) ->
-            ContextThemeWrapper(configuredContext, typedValue.resourceId)
-        else -> ContextThemeWrapper(configuredContext, R.style.StreamUiTheme)
+        this.theme.resolveAttribute(R.attr.streamUiValidTheme, typedValue, true) ->
+            ContextThemeWrapper(this, R.style.StreamUiEmptyTheme)
+        this.theme.resolveAttribute(R.attr.streamUiTheme, typedValue, true) ->
+            ContextThemeWrapper(this, typedValue.resourceId)
+        else -> ContextThemeWrapper(this, R.style.StreamUiTheme)
+    }.apply {
+        applyOverrideConfiguration(
+            Configuration().apply { systemUiMode(uiMode)?.let { this.uiMode = it }}
```

Comment:
This new `Configuration` will loss info from the current configuration on the given Context

Reply:
Fixed, changed the constructor to add the original `Configuration`